source/docs/articles/sites/domains/adding-a-ssl-certificate-for-secure-https-communication.md

```diff
@@ -108,23 +108,22 @@ We recommend using an IPv4 address, unless you are familiar with and understand
 
 ## Test SSL
 
-Before you point your DNS to the custom IP address you received after enabling SSL, you may verify that the certificate is correct.
+Before you point your DNS to the custom IP address you received after enabling SSL, you can verify that the certificate is correct.
 
 <div class="alert alert-danger" role="alert">
 <h4>Warning</h4>
 You may see that the SSL certificate matches your intended domain, but do not expect to view the contents of the site, as we use HTTP headers to route your domain correctly.</div>
 
-### Chrome
+### Test with a Browser
```

Comment:
these instructions work with Chrome, but are slightly different for Firefox, for example.